packages/flutter_driver/lib/src/common/message.dart

```diff
@@ -32,6 +32,21 @@ abstract class Command {
   /// Identifies the type of the command object and of the handler.
   String get kind;
 
+  /// Whether this command requires the widget tree to be initialized before
+  /// the command may be run.
+  ///
+  /// This defaults to true to force the application under test to call [runApp]
+  /// before attempting to remotely drive the application. Subclasses may
+  /// override this to return false if they allow invocation before the
+  /// application has started.
+  ///
+  /// See also:
+  ///
+  ///  * [WidgetsBinding.isRootWidgetAttached], which indicates whether the
+  ///    widget tree has been initialized.
+  @protected
```

Comment:
This probably shouldn't be protected if you want to use it in extension.dart?

Reply:
Oops - done